closed/make/copy/Copy-jdk.jfr.gmk

```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
-# (c) Copyright IBM Corp. 2018, 2020 All Rights Reserved
+# (c) Copyright IBM Corp. 2023, 2024 All Rights Reserved
```

Comment:
If this file were staying, the first year should not be changed.

Reply:
Thanks Keith, Addressed.

Reply:
Yeah got it. 

> However, in later versions, it seems that *.jfc files are already handled, and that copying metadata.xml is not needed. 

Yes that's correct. The change is specific to JDK11.

> Here, simply removing closed/make/copy/Copy-jdk.jfr.gmk and allowing make/copy/Copy-jdk.jfr.gmk to be used should be all that is needed.

ok, However the  [make/copy/Copy-jdk.jfr.gmk](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/3d65f3fd583bab8ac4a97b914f5aef47c709e70e/make/copy/Copy-jdk.jfr.gmk) includes `CopyCommon.gmk` and also I believe it should be wrapped within OPENJ9_ENABLE_JFR check. 

Could you please kindly let me know what is the recommended approach?

Reply:
The entire `jdk.jfr` module is omitted if `OPENJ9_ENABLE_JFR` is false; see [`closed/custom/common/Modules.gmk`](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/openj9/closed/custom/common/Modules.gmk#L46), which means the related makefile fragments (including `make/copy/Copy-jdk.jfr.gmk`) are irrelevant. The only change that is needed is to remove `closed/make/copy/Copy-jdk.jfr.gmk`.